github/common.js

```diff
@@ -113,6 +113,21 @@ function augment(a) {
   }
 }
 
+function augmentTooltip(a) {
+  const authorElement = document.querySelector("a.author.pull-header-username");
+  const who = a.getAttribute("aria-label");
```

Comment:
nit: `tooltip` or `label` is probably more descriptive.

Reply:
Indeed. I just copied this from above.